com.unity.netcode.gameobjects/Tests/Runtime/AssemblyInfo.cs

```diff
@@ -0,0 +1,5 @@
+using System.Runtime.CompilerServices;
```

Comment:
Why are we using internalsvisibleto? Shouldn't we just make those things public? I mean, it's a test assembly...

Reply:
I could be wrong about why Josie did it this way but Package Verification was throwing warnings about undocumented public APIs for test assemblies in the tools package, so we made them all internal and used exposed internals for cross-assembly references to address that.

Reply:
The classes in the test assembly make use of internal classes from the tools/NGO repos, so when I made the test classes public there was then a mismatch between the levels of accessibility. So this seemed the best way around the issue.